docs/guide/coverage.md

```diff
@@ -70,6 +70,55 @@ export default defineConfig({
 })
 ```
 
+## Custom Coverage Reporter
+
+You can use custom coverage reporters by passing either name of the package or absolute path in `test.coverage.reporter`:
+
+```ts
+// vitest.config.ts
+import { defineConfig } from 'vitest/config'
+
+export default defineConfig({
+  test: {
+    coverage: {
+      reporter: [
+        // Specify reporter using name of the NPM package
+        ['@vitest/custom-coverage-reporter', { someOption: true }],
+
+        // Specify reporter using local path
+        '/absolute/path/to/custom-reporter.cjs',
+      ],
+    },
+  },
+})
+```
+
+Custom reporters are loaded by Istanbul and must match its reporter interface. See [built-in reporters' implementation](https://github.com/istanbuljs/istanbuljs/tree/master/packages/istanbul-reports/lib) for reference.
```

Comment:
shouldn't this be wrapped in some sort of info card?

Reply:
As it's describing the code block below it I think it should be as is, and not wrapped in `:::tip` card.